app/client/templates/views/dashboard.js

```diff
@@ -18,8 +18,8 @@ Template['views_dashboard'].helpers({
 
     @method (wallets)
     */
-    'wallets': function(disabled){
-        var wallets = Wallets.find({disabled: disabled}, {sort: {creationBlock: 1}}).fetch();
+    'wallets': function(){
+        var wallets = Wallets.find({disabled: false}, {sort: {creationBlock: 1}}).fetch();
```

Comment:
Or better this needs to be reverted, as well as in the `dashboard.html`, and we need to make sure the wallet get the disabled property to `true` or `false`.